src/style.css

```
@@ -0,0 +1,87 @@
+* {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.snake {
+  line-height: 300px;
+  min-width: 300px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: #fff;
+  text-align: center;
+  flex-grow: 1;
+  flex-basis: 33.33%;
+}
+
+.container :nth-child(1) {
+  background-color: rgb(255, 0, 0);
+}
+
+.container :nth-child(2) {
+  background-color: rgb(204, 0, 0);
+}
+
+.container :nth-child(3) {
+  background-color: rgb(153, 0, 0);
+}
+
+.container :nth-child(4) {
+  background-color: rgb(102, 0, 0);
+}
+
+.container :nth-child(5) {
+  background-color: rgb(51, 0, 0);
+}
+
+.container :nth-child(6) {
+  background-color: rgb(0, 0, 0);
+}
+
+@media screen and (min-width: 600px) and (max-width: 899px) {
```

```suggestion
@media screen and (min-width: 600px) {
```
Можно не писать, следующий медиазапрос перекроет нужные стили